browse/config.py

```diff
@@ -161,36 +163,6 @@
 available. This defaults to http.
 """
 
-JSON_AS_ASCII = os.environ.get("JSON_AS_ASCII") == ON
```

Comment:
These configs are no longer supported by Flask. It appears they were not in use anyway.